packages/webview_flutter/webview_flutter_platform_interface/lib/src/platform_webview_controller.dart

```diff
@@ -318,7 +318,7 @@ abstract class PlatformWebViewController extends PlatformInterface {
   /// Sets a callback that notifies the host application that the web page
   /// wants to display a JavaScript prompt() dialog.
   Future<void> setOnJavaScriptTextInputDialog(
-      Future<String> Function(JavaScriptTextInputDialogRequest request)
+      Future<String?> Function(JavaScriptTextInputDialogRequest request)
```

Comment:
This is a breaking change; we'll need to make a replacement method instead (e.g., `setOnJavaScriptTextInputDialogCancelable`).

We should also document what `null` means in its doc comment.

Reply:
Is there not a way to submit this without a breaking change? The Android and iOS implementation PRs have not landed yet.

Reply:
It's already breaking our own tests in other packages to try to change it, and there's no reason a third-party implementation of `webview_flutter` couldn't already be relying on it.

Reply:
Thats fair. I'll create a separate method for it. It's unfortunate that I didn't catch this earlier.

Reply:
Yes, the reason I set the process up to have the individual PRs only split out once the combined PR has been fully reviewed is to try to avoid this kind of thing, but it's inevitable that sometimes we just won't notice things until that final read-through.